telemetry/roles/orchestrator/tasks/deploy_k8s.yml

```diff
@@ -34,7 +33,7 @@
 
 - name: Untar kubespray git repo
   ansible.builtin.unarchive:
-    src: "{{ kubespray_path }}/kubespray.tar.gz"
+    src: "{{ kubespray_path }}/{{ telemetry_package_map['kubespray'] }}.tar.gz"
```

Comment:
You missed to do the update on line 47 and line 51
 it should be:
ansible-playbook {{ kubespray_path }}/{{ telemetry_package_map['kubespray'] }}/cluster.yml \

Reply:
Addressed changes in internal PR. It will be available in next set of PRs